cmake/features.cmake

```diff
@@ -112,7 +112,8 @@ ie_option (ENABLE_IR_V7_READER "Enables IR v7 reader" ${ENABLE_IR_V7_READER_DEFA
 
 ie_option (ENABLE_GAPI_PREPROCESSING "Enables G-API preprocessing" ON)
 
-ie_option (ENABLE_MULTI "Enables Multi Device Plugin" ON)
+ie_option (ENABLE_MULTI "Enables MULTI Device Plugin" ON)
+ie_option (ENABLE_AUTO "Enables AUTO Device Plugin" ON)
```

Comment:
why do we need 2 options? AUTO == MULTI

Reply:
I propose to have one optionwhich will be aligned with the plugin name

Reply:
@ilya-lavrenov @ilyachur 
Just for clarification, AUTO device and MULTI device are different application interface while they are supported by one plugin so file. 
End user may select one of  AUTO device and MULTI device. Then we expect OV still build plugin so file and only expose one device name in plugins.xml.